auk-notebook.ipynb

```diff
@@ -706,8 +807,9 @@
    "source": [
     "# Ego network for a particular node.\n",
     "\n",
-    "largest_node = sorted(graph.nodes('Degree'), key=lambda s: s[1], reverse=True)[0][0] # [1][0] is second largest, etc\n",
-    "neigh = graph.subgraph(graph.neighbors(largest_node))\n",
+    "largest_node = sorted(graph.nodes('Degree'), key=lambda s: s[1], reverse=True)[NETWORK_EGO_NODE][0] # [1][0] is second largest, etc\n",
+    "node = NETWORK_EGO_NODE_SPECIFIC if (NETWORK_EGO_NODE_SPECIFIC) else largest_node\n",
```

Comment:
When I ran this cell I got

```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-33-3411e2be5e3d> in <module>()
      1 # Ego network for a particular node.
      2 
----> 3 largest_node = sorted(graph.nodes('Degree'), key=lambda s: s[1], reverse=True)[NETWORK_EGO_NODE][0] # [1][0] is second largest, etc
      4 node = NETWORK_EGO_NODE_SPECIFIC if (NETWORK_EGO_NODE_SPECIFIC) else largest_node
      5 neigh = graph.subgraph(graph.neighbors(node))

NameError: name 'NETWORK_EGO_NODE' is not defined
```

Reply:
This should be fixed in the latest commit.